ci/build.py

```diff
@@ -248,7 +383,11 @@ def load_docker_cache(tag, docker_registry) -> None:
     else:
         logging.info('Distributed docker cache disabled')
 
+
 def main() -> int:
+    import pprint as pp
```

Comment:
Should we have it where all other imports are?

Reply:
I plan to remove this, is just for diagnostics. TODO.